src/css/forms.scss

```diff
@@ -65,7 +65,7 @@
 }
 
 .form__label {
-    font-size: 12px;
+    font-size: 13px;
```

Comment:
this impacts all the pages I assume. Is this intentional?

Reply:
yes